src/avfoundation.cs

```diff
@@ -11481,7 +11481,14 @@ interface AVCaptureDeviceFormat {
 
 		[iOS (8,0), NoMac]
 		[Export ("videoHDRSupported")]
+		bool VideoHDRSupported { [Bind ("isVideoHDRSupported")] get; }
```

Comment:
```suggestion
		bool IsVideoHdrSupported { [Bind ("isVideoHDRSupported")] get; }
```
The rest of the api uses Hrd instead of HRD see the [gist](https://gist.github.com/7f8a4574716529a18aef210129aac106). The also use the verb "Is"